---
layout: post
title: "Nuxtで、Errorを発火させた後の処理も実行されてしまう"
date: 2019-11-19 02:22:56
categories: vue.js nuxt.js
---
<p>以下の処理を書いた際、after errorというログが表示されます。<br>
エラー後の処理を実行されたら困るのですが、何か対処方法はありますでしょうか？    </p>

<pre><code>async asyncData({ app, error }) {

    error({
        statusCode: 404,
        message: '存在しないURLです'
    })

    console.log("after error")

},
</code></pre>
